microcosm_eventsource/stores/rollup.py

```diff
@@ -130,6 +135,13 @@ def _search_query(self, aggregate=None, **kwargs):
             **kwargs
         )
 
+        if offset is not None:
```

Comment:
I just put this logic into `microcosm-flask` `1.9.1` as a helper function called `Store._paginate`

Reply:
I'm looking through the lib but not sure where this logic is. Could you please reference

Reply:
https://github.com/globality-corp/microcosm-postgres/blob/b3f2ee5f9bcc7425a210108d9702270d108132e5/microcosm_postgres/store.py#L228